models.py

```diff
@@ -0,0 +1,77 @@
+"""
```

Comment:
this whole file can be removed (or the contents replaced w/ something more relevant)

Reply:
I plan add to this rather than remove because it is a good example plus maybe we would like to use a CNN in the future. Although, if it is becoming messy and too long I'll remove the unused parts.

Reply:
I doubt we will have any reason to use a CNN.